### PR TITLE
Updates and corrections to Adobe-generated alert policy info

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Update:  Adobe-generated alert policies are also available on Starter Production environments
- Updated the "Manage disk space" and "Health notifications" topics with link to information about using Adobe-generated alert policies to monitor performance.
- Fixed notification channel instructions to remove specific example about Slack
- Editorial fixes

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/cloud/project/manage-disk-space.html
https://devdocs.magento.com/guides/v2.3/cloud/integrations/health-notifications.html
https://devdocs.magento.com/guides/v2.3/cloud/project/new-relic.html

### CODE REVIEW

File: guides/v2.2/cloud/project/new-relic.md [hunk 1]
```diff
@@ -184,9 +184,9 @@ Prerequisites
 {:.procedure}
 To review Adobe-generated alert policies:
 
-1. Use your Project Owner New Relic credentials to [log in to New Relic](https://login.newrelic.com/login).
+1. Use your Project Owner New Relic credentials to [log in to your New Relic account](https://login.newrelic.com/login).
 
-1. From the navigation menu, select  **Alerts**  >  **Alert Policies**.
+1. From the navigation  menu, select  **Alerts**  >  **Alert Policies**.
```
Comment (keharper) [inline guides/v2.2/cloud/project/new-relic.md]:
```suggestion
1. From the navigation menu, select  **Alerts**  > **Alert Policies**.
```

File: guides/v2.2/cloud/project/new-relic.md [hunk 2]
```diff
@@ -164,13 +164,13 @@ You can learn more about using the New Relic APM and Infrastructure agents to co
 
 ## Monitor performance with alert policies
 
-The New Relic service for {{ site.data.var.ece }} Pro Production environments includes a set of Adobe-generated alert policies to track the following performance metrics:
+Adobe has generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```
Comment (meker12) [inline guides/v2.2/cloud/project/new-relic.md]:
```suggestion
Adobe provides a generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

Comment (meker12) [inline guides/v2.2/cloud/project/new-relic.md]:
```suggestion
Adobe provides a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

File: guides/v2.2/cloud/project/new-relic.md [hunk 3]
```diff
@@ -164,13 +164,13 @@ You can learn more about using the New Relic APM and Infrastructure agents to co
 
 ## Monitor performance with alert policies
 
-The New Relic service for {{ site.data.var.ece }} Pro Production environments includes a set of Adobe-generated alert policies to track the following performance metrics:
+Adobe provides a generated a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```
Comment (meker12) [inline guides/v2.2/cloud/project/new-relic.md]:
```suggestion
Adobe provides a set of New Relic alert policies for {{ site.data.var.ece }} Pro and Starter Production environments to track the following key performance metrics:
```

**Comment by meker12** [comment]
running tests

**Pull request merged**